gnosis/safe/addresses.py

```diff
@@ -830,4 +830,9 @@
             739882,
         )
     ],
+    EthereumNetwork.ZORA: [
+        ("0x3E5c63644E683549055b9Be8653de26E0B4CD36E", 11932, "1.3.0+L2"),
+        ("0xd9Db270c1B5E3Bd161E8c8503c55cEABeE709552", 11934, "1.3.0"),
+        ("0xa6B71E26C5e0845f74c812102Ca7114b6a896AB2", 11914),
```

Comment:
Why this contract?

Reply:
This contract is the safe factory proxy address `GnosisSafeProxyFactory` – does the indexer need to know that?

Reply:
See the contract here: https://explorer.zora.energy/address/0xa6B71E26C5e0845f74c812102Ca7114b6a896AB2

Reply:
If you take a look at the file you will see there's a different section for Proxy factory addresses

Reply:
My bad – fixed.